src/openrct2/GameState.cpp

```diff
@@ -73,6 +76,7 @@ void GameState::InitAll(int32_t mapSize)
 
 void GameState::Update()
 {
+    static uint32_t pctUpdates = 0;
```

Comment:
Static variables won't work well when we want to support having multiple game-states in the future.